4-projects/README.md

```diff
@@ -94,3 +94,14 @@ When using Cloud Build or Jenkins as your CI/CD tool each environment correspond
 1. Run `./tf-wrapper.sh apply development`
 
 If you received any errors or made any changes to the Terraform config or `terraform.tfvars` you must re-run `./tf-wrapper.sh plan <env>` before run `./tf-wrapper.sh apply <env>`
+
+### Optionals
+
+This module provides some optional projects configurations. You can check below what are they and what will be created if you enable each one:
+
```

Comment:
We should remove these and auto generate as per feedback here:

https://github.com/terraform-google-modules/terraform-example-foundation/pull/239